shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -519,6 +519,10 @@ public AccessibilityNodeInfo createAccessibilityNodeInfo(int virtualViewId) {
         result.setClassName("android.view.View");
         result.setSource(rootAccessibilityView, virtualViewId);
         result.setFocusable(semanticsNode.isFocusable());
+        if (semanticsNode.isFocusable()) {
```

Comment:
Should we only do that if the framework explicitly tells us that the node is focusable via the flag? "isFocusable" uses a hacky heuristic because previously we didn't have that information available.

Reply:
Well, but this is where I'm going to connect up the Focus widget so that it sets the focusable bit on the semantics node it creates.  Then it won't be so hacky.